### PR TITLE
Added Litestream

### PR DESCRIPTION
**Pull request by sandro-lombardi**
Hi there. I'd like to add [Litestream](https://litestream.io/). 

It's a streaming replication tool for SQLite and at the time of writing at 8.3k stars on [github](https://github.com/benbjohnson/litestream) and 1M+ pulls on [dockerhub](https://hub.docker.com/r/litestream/litestream).

Thank you in advance for your time in considering this PR!

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] I've executed the checks if necessary by running `npm ci && npm run validate_apps && npm run formatter`

### CODE REVIEW

File: public/v4/apps/litestream.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            - $$cap_appname-data:/data
+            - $$cap_appname-config:/config
+        environment:
+            LITESTREAM_ACCESS_KEY_ID: $$cap_access_key_id
+            LITESTREAM_SECRET_ACCESS_KEY: $$cap_secret_access_key
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: Litestream Version
+          defaultValue: 'sha-1dbd8615908b1d3cebbfc44eb40a35056e15ba4c'
```
Comment (githubsaturn) [inline public/v4/apps/litestream.yml:23]: Why not using one of their stable versions? 0.3 for example

File: public/v4/apps/litestream.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/litestream/litestream/tags
+          validRegex: "/^([^\\s^\\/])+$/"
+        - id: '$$cap_access_key_id'
+          label: Access key ID
+          defaultValue: ''
```
Comment (githubsaturn) [inline public/v4/apps/litestream.yml:28]: Can we use `$$cap_gen_random_hex(<LENGTH>)` for the access key and secret?

File: public/v4/apps/litestream.yml (hunk 1 above)
Comment (sandro-lombardi) [reply public/v4/apps/litestream.yml]: Absolutly, I missed that. Changed it to 0.3.

File: public/v4/apps/litestream.yml (hunk 2 above)
Comment (sandro-lombardi) [reply public/v4/apps/litestream.yml]: I've added `$$cap_gen_random_hex` for both, the access key and the secret. For the length, I choose 20 and 40 respectively, which I believe correspond to the AWS S3 access key and secret key length. For other S3 compatible storage providers, this length might differ (e.g. Storj.io uses 28 and 53).

**Comment by githubsaturn** [comment]
Thanks!

File: public/v4/apps/litestream.yml (hunk 1 above)
Comment (geoman2) [reply public/v4/apps/litestream.yml]: Hi, the 0.4 version is released.

**Comment by jdkdev** [comment]
Awesome thanks @sandro-lombardi

**Pull request merged**